### ISSUE TITLE
Provide more simple examples

### ISSUE THREAD

**Issue by ralight** [eclipse/dash-licenses#60]
All of the examples provided in the documentation are in the form `magic-dependency-command | java -jar org.eclipse.dash.licenses-<version>.jar -`. Could there please be some simple examples of what needs to be sent to the jar, for projects that don't have a magic dependency command? It took me a while to figure out the syntax for checking something on github is `git/github/eclipse/mosquitto/2.0.9`, for example.

**Comment by waynebeaton** [comment]
Of course. Can you give me a more complete example of what you ended up throwing at the license checker?

**Comment by ralight** [comment]
The commands I tried were all manual and github related (it was the syntax that eluded me):

```
echo git/github/eclipse/mosquitto/2.0.9 | java -jar org.eclipse.dash.licenses-SNAPSHOT.jar -
echo git/github/DaveGamble/cJSON/1.7.14 | java -jar org.eclipse.dash.licenses-SNAPSHOT.jar -
echo git/github/warmcat/libwebsockets/4.0.0 | java -jar org.eclipse.dash.licenses-SNAPSHOT.jar -
```
